muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/workspaces/index.ts

```diff
@@ -373,13 +375,13 @@ export interface MaterialAssignmentType {
   assignmentType: "EXERCISE" | "EVALUATED",
   correctAnswers: string,
   path: string,
-  title: string
+  title: string,
```

Comment:
Why comma after last element?

Reply:
Normally it's preferred to be a ";" but when we developed muikku using eclipse we used commas because there was no style preference then.

The purpose of adding a , at the end is that when diffing later if you add one last element, the line where a comma exists will not be affected

{
  propertyA: string,
  propertyB: string,
}

Then we need to introduce propertyC

{
  propertyA: string,
  propertyB: string,
  propertyC: string,
}

Now when passing through diffing tool only propertyC line will be highlighted as the only difference.

If there wasn't a comma this wouldn't happen; that's the reason its' preferred; now it should usually be ";" but we used some old style when making muikku when we used eclipse then.